Description: Pass the correct compiler to the OpenJDK build system
Author: Matthias Klose <doko@ubuntu.com>
Forwarded: #openjdk on OFTC

--- a/Makefile.am
+++ b/Makefile.am
@@ -617,6 +617,7 @@
 	VERBOSE="$(VERBOSE)" \
 	STATIC_CXX="false" \
 	BUILD_GCC="$(CC)" \
+	BUILD_CPP="$(CXX)" \
 	BUILD_CXX="$(CXX)" \
 	SYSTEM_CUPS="true" \
 	CUPS_LIBS="${CUPS_LIBS}" \
